docs/.vuepress/public/presets/toml/pastel-powerline.toml

```diff
@@ -0,0 +1,69 @@
+format = """
+[](#9A348E)\
+$username\
+[](bg:#DA627D fg:#9A348E)\
+$directory\
+[](fg:#DA627D bg:#FCA17D)\
+$git_branch\
+$git_status\
+[](fg:#FCA17D bg:#86BBD8)\
+$nodejs\
+$rust\
+[](fg:#86BBD8 bg:#33658A)\
+$time\
+[ ](fg:#33658A)\
+"""
+
+# Disable the blank line at the start of the prompt
+# add_newline = false
+
+# You can also replace your username with a neat symbol like  to save some space
+[username]
+show_always = true
+style_user = "bg:#9A348E"
+style_root = "bg:#9A348E"
+format = '[$user ]($style)'
+
+[directory]
+style = "bg:#DA627D"
+format = "[ $path ]($style)"
+truncation_length = 3
+truncation_symbol = "…/"
+
+# Here is how you can shorten some long paths by text replacement
+# similar to mapped_locations in Oh My Posh:
+[directory.substitutions]
+"Documents" = " "
+"Downloads" = " "
+"Music" = " "
+"Pictures" = " "
+# Keep in mind that the order matters. For example:
+# "Important Documents" = "  "
+# will not be replaced, because "Documents" was already substituted before.
+# So either put "Important Documents" before "Documents" or use the substituted version:
+# "Important  " = "  "
+
+[git_branch]
+symbol = ""
+style = "bg:#FCA17D"
+format = '[[ $symbol $branch ](bg:#FCA17D)]($style)'
+
+[git_status]
+style = "bg:#FCA17D"
+format = '[[ $all_status$ahead_behind ](bg:#FCA17D)]($style)'
```

Comment:
In testing locally, I noticed there was some extra padding around the git status, which is especially visible when there's no status indicator.

<img width="1439" alt="CleanShot 2022-03-17 at 17 47 51@2x" src="https://user-images.githubusercontent.com/4658208/158906714-c1c10b3e-0943-4dfa-8256-0624c6083ed2.png">

<img width="1074" alt="CleanShot 2022-03-17 at 17 49 27@2x" src="https://user-images.githubusercontent.com/4658208/158906864-aa5c5931-f795-471f-90ad-2a8614529235.png">

Parentheses will only show their contents when variables within them have values, so this change will remove any excess space when there's no status 😄 

```suggestion
format = '[[($all_status$ahead_behind )](bg:#FCA17D)]($style)'
```